Solutions/README.md

```diff
@@ -1 +1,122 @@
+  Workbooks?: string[];
+  "Analytic Rules"?: string[];
+  Playbooks?: string[];
+  Parsers?: string[];
+  "Hunting Queries"?: string[];
+  "Data Connectors"?: string[];
+  BasePath?: string; //Optional base path to use. Either Internet URL or File Path. Default = "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/""
+}
+```
+
+**Example of Input File: Solution_McAfeePO.json**
+```json
+{
+  "Name": "McAfeePO",
+  "Author": "Amarnath Pamidi – v-ampami@microsoft.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/McAfeeePO/Workbooks/Images/Logo/mcafee_logo.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "The [McAfee ePO](https://www.mcafee.com/enterprise/en-in/products/epolicy-orchestrator.html) is a centralized policy management and enforcement for your endpoints and enterprise security products. McAfee ePO monitors and manages your network, detecting threats and protecting endpoints against these threats.",
+  "WorkbookDescription": "Gain insights into McAfeePO logs.",
+  "Workbooks": [
+    "Workbooks/McAfeeePOOverview.json"
+  ],
+  "Analytic Rules": [
+    "Analytic%20Rules/McAfeeEPOAgentHandlerDown.yaml",
+    "Analytic%20Rules/McAfeeEPOAlertError.yaml"
+  ],
+  "Parsers": [
+    "Parsers/McAfeeEPOEvent.txt "
+  ],
+  "Hunting Queries": [
+    "Hunting%20Queries/McAfeeEPOAgentErrors.yaml"
+  ],
+  "Data Connectors": [
+    "Data%20Connectors/Connector_McAfee_ePO.json"
+  ],
+  "BasePath": "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions"
+}  
+```
+
+## Generate Solution Package
+Run generateSolution script in the automation folder which will create solution package with createUIDefinition.json, mainTemplate.json and zip file(version (1.0.0) as name) in the solutions folder with respect to the given input file. For every new modification to the files createUIDefinition.json and mainTemplate.json, a new zip file containing modified files should be created with latest version name (1.0.1, 1.0.2…..) and should be placed in the solution folder along with previous zip file.
+
+## Validate Package
+Validate the Solution offer files createUiDefinition.json and mainTemplate.json using the Azure Toolkit / TTK CLI tool. This tool helps with static analysis. User can validate package through two methods.
+
+**Through generated xml files:**
+1.	Go to path C:\One\Sentinel-AzMP-Solutions\tmp\results\arm-ttk
+2.	Check the xml files of respective solution to check the validations.
+3.	If errors are displayed make necessary modifications to the files so it passes all validations.
+
+**Through commands:**
+
+**Setup & Run arm-ttk tool:**
+1.	Clone the repository https://github.com/Azure/arm-ttk to C:\One 
+    *	If C:\One does not exist, create the folder. 
+    * You may also choose a different folder but reference it properly while importing module.
+1.	Open powershell, go to the solution path and run the following command to import arm-ttk module.
```

Comment:
If they do this they will have to import it every single time.
They must instead add it to their profile.
Example command to add to profile: `Add-Content -Path $Profile -Value "Import-Module C:\One\arm-ttk\arm-ttk\arm-ttk.psd1"`

Reply:
Updated the content.